nautobot/core/templates/inc/nav_menu.html

```diff
@@ -28,9 +28,7 @@
                     class="nb-sidenav-list-item"
                     type="button"
                 >
-                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-                        <circle cx="12" cy="12" fill="none" r="10" stroke="currentcolor" stroke-width="1.5"/>
-                    </svg>
+                    <img alt="star" src="{% static 'icon/star.svg' %}">
```

Comment:
I'm worried about `icon` static directory name potentially colliding with something, can we use something like `nautobot-icons` for safety?